docs/access.rst

```diff
@@ -31,6 +31,19 @@ While it is running, **Kolibri** will display an icon in the Windows taskbar (us
 .. tip:: 
   If you close the browser window, Kolibri will still be running in the background. Use the **Load in browser** menu item to reopen it in the browser.
 
+
+.. _ncomputing: 
+
+.. warning::
+  If you install Kolibri in settings with various virtual desktops running on the same single hardware device (for example with NComputing), you need to make sure that Kolibri is running **only from the admin account**. Confirm that the **Run Kolibri at system startup** taskbar option is checked only when the admin logs in, but unchecked when any other user of virtual desktops is logged in.
```

Comment:
```suggestion
  In some Windows multi-user environments (for example NComputing), you need to make sure that Kolibri is running **only from the admin account** to avoid having Kolibri started for each login/session. Confirm that the **Run Kolibri at system startup** taskbar option is checked only when the admin logs in, but unchecked when any other user of virtual desktops is logged in.
```